ChangeLog.d/session-cache-api.txt

```diff
@@ -0,0 +1,5 @@
+API Changes
```

Comment:
The CI points out that the ChangeLog assembly script wants a lowercase `c` here.

Reply:
Fixed.